spyder/plugins/editor/widgets/tests/test_editor_and_outline.py

```diff
@@ -138,7 +138,80 @@ def test_close_a_file(editorstack, outlineexplorer, test_files):
     # tree has been updated.
     editorstack.close_file(index=1)
     results = [item.text(0) for item in treewidget.get_visible_items()]
-    assert results == ['foo1.py', 'text1.txt']
+    assert results == ['foo1.py', 'foo2.py']
+
+
+def test_sort_file_alphabetically(editorstack, outlineexplorer, test_files):
+    """
+    Test that the option to sort the files in alphabetical order in the
+    outline explorer is working as expected.
+
+    This feature was introduced in PR #8015
+    """
+    editorstack = editorstack(test_files)
+    treewidget = outlineexplorer.treewidget
+    results = [item.text(0) for item in treewidget.get_visible_items()]
+    assert results == ['foo1.py', 'text1.txt', 'foo2.py']
+
+    # Set the option to sort files alphabetically to True and assert the
+    # results.
+    treewidget.toggle_sort_files_alphabetically(True)
+    results = [item.text(0) for item in treewidget.get_visible_items()]
+    assert results == ['foo1.py', 'foo2.py', 'text1.txt']
+
+
+def test_sync_file_order(editorstack, outlineexplorer, test_files):
+    """
+    Test that the order of the files in the Outline Explorer is updated when
+    tabs are moved in the EditorStack.
+
+    This feature was introduced in PR #8015
+    """
+    editorstack = editorstack(test_files)
+    treewidget = outlineexplorer.treewidget
+    results = [item.text(0) for item in treewidget.get_visible_items()]
+    assert results == ['foo1.py', 'text1.txt', 'foo2.py']
+
+    # Invert tab 1 with tab 2 and assert the results.
```

Comment:
"Assert the results"?

Reply:
Are you suggesting that I remove "Assert the results" from the comments? I do not understand what you are suggesting.